src/logservice/logminer/ob_log_miner_analyzer.cpp

```diff
@@ -228,6 +230,7 @@ void ObLogMinerAnalyzer::run1()
               "Writer", writer_task_cnt, "ResourceCollector", rc_task_cnt);
         }
       }
+      usleep(LOOP_INTERVAL);
```

Comment:
better use `ob_usleep`, can you also change `usleep` to `ob_usleep` in `ObLogMinerDataManager`?

Reply:
sure，I will make it in the next commit.